### DISCUSSION TITLE
In-Depth-Laravel-2023-Module-1 Discussions!

### DISCUSSION THREAD

**Discussion by sarthaksavvy** [bitfumes/laravel-10-for-beginners#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of Bitfumes. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## Warning
   * Do not spam
   * Do not write hate speech or words
   * Do not share unrelated link, content, video or image
   
  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phynalZERO** [comment]
Hello! 

I apologize in advance, but I am new to everything here including Github, Laravel, etc..
I am not even sure if this is the right place to ask my question. 
So please correct me if I am wrong.

My name is Nick and I love TRYING to learn how to code. 
I found the Bitfumes videos on Youtube and loved them. 
Found the link for the course and bought it immediately. 
I am struggling through for sure, but still enjoying every minute. 

Right now I am stuck with the Github login. I am getting an error as follows:
"Laravel \ Socialite \ Two \ InvalidStateException"
I am at 6:30 in video 31. 026 - login with GitHub part 2.

Please see attached screenshots.
Let me know what other info I can provide if needed. 
Thank you in advance!

![video-location](https://github.com/bitfumes/In-Depth-Laravel-2023-Module-1/assets/80796032/1d79e43c-550c-44cd-b8b7-f0f0036daeb8)
![2023-10-06 18 27 52](https://github.com/bitfumes/In-Depth-Laravel-2023-Module-1/assets/80796032/c14f6c47-d6b4-4341-a06b-36d71fb6b3f6)
![2023-10-06 18 27 53](https://github.com/bitfumes/In-Depth-Laravel-2023-Module-1/assets/80796032/179c10f2-c9b3-418e-ac42-71f4c65a7a56)

**Comment by sarthaksavvy** [comment]
Hii, I can see that you have created a post route for 'auth/redirect'
May be try to make it a get request
